parser/parser.y

```diff
@@ -663,6 +664,7 @@ import (
 %left   tableRefPriority
 %precedence lowerThanOn
 %precedence on
+%left	assignmentEq
```

Comment:
The assign op should be right instead of left. But the rule make it unnecessary. I think we should keep it.